lib/internal/Magento/Framework/Serialize/Serializer/Json.php

```diff
@@ -34,7 +34,13 @@ public function serialize($data)
      */
     public function unserialize($string)
     {
+        if ($string === null) {
+            throw new \InvalidArgumentException(
+                sprintf('Unable to unserialize value. Error: Parameter must be a string type, null given.')
```

Comment:
`sprintf` not needed here.

Reply:
Hello @xmav, thanks for your review, I fixed code by your recommendation, please check if everything is well.

Thank you